pkg/mimir/modules.go

```diff
@@ -132,7 +133,7 @@ func (t *Mimir) initActivityTracker() (services.Service, error) {
 
 	entries, err := activitytracker.LoadUnfinishedEntries(t.Cfg.ActivityTracker.Filepath)
 
-	l := util_log.Logger
+	l := log.With(util_log.Logger, "module", "activity-tracker")
```

Comment:
[non blocking] In other places we call it "component" when we expose it to user (e.g. metrics or logs). WDYT if we call it "component" here as well?

```suggestion
	l := log.With(util_log.Logger, "component", "activity-tracker")
```

Reply:
Sure, consistency ftw.